### PR TITLE
Bots section revised and edited

### LINKED ISSUE #40: How to read original form messages in Bot
@ValeMP Es posible obtener los mensajes originales como contexto del bot activando la opción "save from response"

Quedan disponibles en el campo { ..., messages }, donde messages es un arreglo de COTMessages 
y para acceder a ellos $VALUE#messages 

image

@guillevalin Tenemos que mejorar el nombre de este input.
@Mauriceac Debemos que actualizar la documentación con este input.

_Originally posted by @acedward in https://github.com/Cotalker/documentation/discussions/39#discussioncomment-576470_

### PR DESCRIPTION
**Pull request by Mauriceac**
- Bots section revised and edited.
- New images added.
- "Save form response" included.

closes #40

### CODE REVIEW

**Comment by Mauriceac** [comment]
## This is the response to the initial issue (Please check if it's ok.):

Screen Shot 2021-06-16 at 4 24 47 PM

File: docs/documentation/admin/admin_bots.md [hunk 1]
```diff
@@ -1,58 +1,158 @@
...
+
+</div>
+<div className="col col--4"><em>This field only appears when the 'It's a survey command' option is activated.</em></div>
+</div>
+
+<div className="row table-row-2">
+<div className="col col--3"><b>Save form response:</b></div>
+<div className="col col--5">
+
+Saves the information sent through the _survey_ as a [COTMessage](/docs/documentation/api/communication/messages) in its [context](/docs/documentation/automation/triggers_and_contexts#channel-survey-trigger). You can access the data with COTLang script, e.g., `$VALUE#messages`.
```
Comment (acedward) [inline docs/documentation/admin/admin_bots.md:127]: @Mauriceac 

If `saves the information` is set, then the messages (COTMessage Array) are **added** to the context response as `messages`, and can be read with COTLang with `$VALUE#messages`. 

NOTE: This feature is deprecated, now messages retrieved by `form.id` can be obtained by comparing this with  `answer.formId`

Comment (Mauriceac) [reply docs/documentation/admin/admin_bots.md]: ## Review: @acedward 
- Edited according to latest indications.
- **Question:** Where can I send users to find more information about *form.id* and  *answer.formId*.
Screen Shot 2021-07-05 at 2 56 24 PM

Comment (acedward) [reply docs/documentation/admin/admin_bots.md]: @Mauriceac This isn't explained anywhere now.

1. We should update the User Model with user.form description
`message.form.id` contains the main-survey id, unique for the entire survey
`message.form.subid` contains the sub-survey id, unique per-sub-survey - and null for the non-sub-survey parts.
 
2. We should add where we introduce Surveys, that each one is built by messages, and these fields are automatically set.

Comment (acedward) [reply docs/documentation/admin/admin_bots.md]: 3. We should update Answer model with `answer.formid` This is a copy of the messages `message.form.id` value

Comment (Mauriceac) [reply docs/documentation/admin/admin_bots.md]: > 1. We should update the Message Model with message.form description...

Screen Shot 2021-07-07 at 5 43 09 PM

> 2. We should add where we introduce Surveys, that each one is built by messages, and these fields are automatically set.

Screen Shot 2021-07-07 at 5 43 48 PM

> 3. We should update Answer model with answer.formId This is a copy of the messages message.form.id value

Screen Shot 2021-07-07 at 5 44 25 PM

**Pull request merged**